Files/Dialogs/SettingsDialog.xaml.cs

```diff
@@ -29,13 +29,22 @@ private void Current_SizeChanged(object sender, WindowSizeChangedEventArgs e)
 
         private void UpdateDialogLayout()
         {
-            if (Window.Current.Bounds.Height <= 600)
+            if (Window.Current.Bounds.Height <= 640)
```

Comment:
Implementation as suggestion:

```suggestion
            if (Window.Current.Bounds.Height <= 710)
```